.github/workflows/git-artifacts.yml

```diff
@@ -0,0 +1,314 @@
+name: git-artifacts
+
+on:
+  workflow_dispatch:
+    inputs:
+      build_only:
```

Comment:
`build_only` sounds like a Boolean. This is totally my fault. But maybe we can fix it "while at it", maybe `artifacts`